modules/custom/az_person/css/az_person.css

```diff
@@ -1,3 +1,14 @@
+/* Display - All */
+.node--type-az-person .field--name-field-az-attachments a {
+	word-break: break-word;
+}
```

Comment:
It looks like this CSS could be replaced with the [text-break class from Bootstrap](https://digital.arizona.edu/arizona-bootstrap/docs/2.0/utilities/text/#word-break). I have only tested in DevTools, but in the display you should be able to create a field group around the CV/Documents field and add the `text-break` class to that.